algorithms/Python/recursion/gcd_using_recursion.py

```diff
@@ -0,0 +1,9 @@
+def gcd(a, b):
```

Comment:
One PR should be preferably restricted to addition of one algorithm only.

Reply:
Ok, I'll make the changes and then commit them. Thank you for reviewing @Arsenic-ATG!